Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -569,10 +569,17 @@ class BrowserViewController: UIViewController {
                                                          object: nil)
     }
 
-    fileprivate func hasPendingCrashReport() -> Bool {
+    fileprivate func crashedLastLaunch() -> Bool {
         return SentryIntegration.crashedLastLaunch
     }
 
+    fileprivate func cleanlyBackgrounded() -> Bool {
+        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else {
+            return false
```

Comment:
I can't speak for Stefan, but I think this is because if we can't get `appDelegate`, something is seriously wrong.